flycheck.el

```diff
@@ -231,11 +231,12 @@ attention to case differences."
     ruby-jruby
     rust-cargo
     rust
-    sass
     scala
     scala-scalastyle
     scheme-chicken
     scss-lint
+    sass/scss-sass-lint
+    sass
```

Comment:
> Am 11.09.2016 um 15:49 schrieb Clément Pit--Claudel <notifications@github.com>:
> 
> In flycheck.el:
> 
> >      scala
> >      scala-scalastyle
> >      scheme-chicken
> >      scss-lint
> > +    sass/scss-sass-lint
> > +    sass
> I think we should keep these sorted alphabetically :)
> 

Not possible here because scss-lint must come first.
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>